server/dns-service/dnsservice-api/src/test/java/org/apache/james/dnsservice/api/mock/DNSFixture.java

```diff
@@ -20,27 +20,31 @@
 
 import java.net.InetAddress;
 import java.net.UnknownHostException;
+import java.util.Collection;
 
 import org.apache.james.dnsservice.api.DNSService;
 
+import com.google.common.collect.ImmutableList;
+
 /**
  * Some DNS Fixtures used by various Test related to DNS and InetNetwork.
  */
 public class DNSFixture {
-
+    public static final String HOST_CANNOT_LOOKUP_BY_NAME = "local";
```

Comment:
I am not sure what you want to mean, don't you want to mean "Unresolvable host" ?
If yes in this case, I think `UNRESOLVABLE_HOST` would be a better naming